app-editors/retext/retext-9999.ebuild

```diff
@@ -25,36 +25,37 @@ if [[ ${PV} == *9999 ]]
 		S="${WORKDIR}"/${MY_P}
 fi
 
-LICENSE="GPL-2"
+LICENSE="GPL-2+"
 SLOT="0"
 IUSE="+spell"
 
 RDEPEND="
+	dev-python/chardet[${PYTHON_USEDEP}]
 	dev-python/docutils[${PYTHON_USEDEP}]
-	dev-python/markdown[${PYTHON_USEDEP}]
-	>=dev-python/markups-2.0[${PYTHON_USEDEP}]
-	>=dev-python/chardet-2.3[${PYTHON_USEDEP}]
+	dev-python/markdown[pygments,${PYTHON_USEDEP}]
```

Comment:
Do not depend on flags that don't do anything. This only asks users to play with `package.use` for no real reason. It's sufficient to depend on pygments which you already do.